metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg_profiler.py

```diff
@@ -181,24 +201,18 @@ def profile_table(
             aspect=dataset_profile,
         ).as_workunit()
 
-    # The following will eventually be done by the Iceberg API (in the new Python refactored API).
     def _renderValue(
```

Comment:
While at it, should we make this proper snake casing?
```suggestion
    def _render_value(
```